azure-pipelines.yaml

```diff
@@ -0,0 +1,61 @@
+name: Xamarin.Android.Tools $(Rev:r)
+variables:
+  major: 1
+  minor: 0
+  patch: $[counter('versioncounter', 100)]
+jobs:
+- job: windows
+  displayName: windows
+  pool: Hosted VS2017
+  steps:
+  - task: NuGetCommand@2
+    displayName: 'NuGet restore'
+    inputs:
+      restoreSolution: Xamarin.Android.Tools.sln
+  - task: MSBuild@1
+    displayName: 'Build solution Xamarin.Android.Tools.sln'
+    inputs:
+      solution: Xamarin.Android.Tools.sln
+  - task: NuGetCommand@2
```

Comment:
Instead of this block, should we instead add the `NUnit3TestAdapter` to a `@(PackageReference)` within `Xamarin.Android.Tools.AndroidSdk.csproj` so that it's restored alongside all the other NuGet dependencies, via the `nuget restore Xamarin.Android.Tools.sln` step?

Reply:
I agree, though I think it would make sense to add it to the test project (`Xamarin.Android.Tools.AndroidSdk-Tests.csproj`) instead.

Reply:
Whichever. :-)